browser/components/MarkdownEditor.js

```diff
@@ -142,6 +142,12 @@ class MarkdownEditor extends React.Component {
     this.renderPreview(this.props.value)
   }
 
+  handleKeyDown(e) {
+    if (this.state.status === 'CODE' && e.key == 'Escape') {
```

Comment:
> Ctrl + E is best.

I agree. I'll try to implement multiple key handler for shortcuts.

> it may be better to make it customizable.

I think so.